frontend/lib/util/filesystem.cpp

```diff
@@ -78,37 +78,28 @@ static std::string getTempDir() {
 #endif
 }
 
-FILE* openfile(const char* path, const char* mode, ErrorMessage& errorOut) {
+FILE* openfile(const char* path, const char* mode, std::string& errorOut) {
   FILE* fp = fopen(path, mode);
   if (fp == nullptr) {
-    std::string strerr = my_strerror(errno);
     // set errorOut. NULL will be returned.
-    errorOut = ErrorMessage::error(Location(),
-                                   "opening %s: %s",
-                                   path, strerr.c_str());
+    errorOut = my_strerror(errno) + ".";
```

Comment:
We are losing the "opening/closing/etc." info with this change, no?

Reply:
Yes -- I believe I did this because the error string already contains that information, so it would say `opening` twice. However I'm unable to come up with an example that will print this error alone facing the user, since user-facing instances of it seem to go through `readfile`.